var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -43,7 +43,6 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkg-config", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
```

Comment:
I would say yes, if it had any benefit whatsoever. But if it doesn't, it makes it harder to test specific versions of dependencies.